test/scripts/gh-actions/run-e2e-tests.sh

```diff
@@ -28,7 +28,7 @@ else
 fi
 
 PARALLELISM="${2:-1}"
-source python/kserve/.venv/bin/activate
+#source python/kserve/.venv/bin/activate
```

Comment:
is this intended?
IIRC the dependencies are installed using this venv.

Reply:
for ci, I didn't check but from the script, it does not setup venv, so it failed on local

Reply:
updated